wiki/Modding/en.md

```diff
@@ -1,262 +1,42 @@
-The artist name, title of the song, source (if applicable), and tags should all be the same in each difficulty.
-
-Here are some other things to look out for in the song setup:
-
-- The `Enable Countdown` checkbox should be either enabled or disabled for all difficulties.
-- The combo colours should be the same for each background per difficulty in a mapset.
-  Guest mappers are an exception, but generally: difficulties by the same person and with the same background should use the same combo colours.
-- Maps with widescreen storyboards should have the setting `Widescreen Support` (in `Design` tab) enabled to avoid being cut off in 16:9 resolutions.
-- Maps with storyboards that can cause epilepsy (frequent strobes over 3Hz or one-third of a second) have to enable `Display epilepsy warning (storyboard has quick strobing)` (in `Design` tab).
-- If a map has [breaks](/wiki/breaks), the setting `Letterbox during breaks` should be consistent throughout a mapset.
-    The `Letterbox during breaks` setting must be disabled for osu!mania difficulties.
-- The `Preferred Skin` setting should be set to `User's preference (No Change)`, this may sometimes set to the mapper's own skin by mistake.
-
-#### Difficulty
-
-*See also: [osu!standard Ranking Criteria](/wiki/osu!standard_Ranking_Criteria#difficulty-specific) (difficulty-specific section)*
-
-The difficulty settings can be found in the song setup dialog.
-These are important to game play so that the song is at a fair level of difficulty for the varying difficulties a mapset can have.
-
-For example, an Easy difficulty does not really need an [approach rate](/wiki/approach_rate) (AR) of 7 or an [HP drain rate](/wiki/HP_drain_rate) (HP) of 8.
-Using these values for an Easy difficulty makes this completely illogical and will make the difficulty much harder than it should be.
-
-### Background, Skin, and Combo Colours
-
-All beatmaps must have a background image.
-The skin you supply should not blend with the background image or greatly hinder (or chellenge) game play.
-For example, complicated hit circle design, huge hit bursts, etc.
+**Modding** is the act of providing feedback on another user's [beatmap](/wiki/Beatmaps). A user who is modding is referred to as a *modder*, and their collective posts on a beatmap's discussion are referred to as a *mod*.
 
-There are also many [skinning](/wiki/skinning) rules that should be followed.
-Similar to skinning, combo colours should not blend in with the background either.
-These combo colours should also be very distinct and clear.
-The mapper should not have two almost identical combo colours unless they are separated by a very contrasting colour in between.
+Mods are mainly used to improve beatmaps aiming for Ranked status. Because it is a detailed and analytical process, modding beatmaps is also a popular way for users improve their understanding of beatmapping.
 
-### Storyboarding
+## Finding beatmaps to mod
```

Comment:
this section should probably start off with a short paragraph saying there are many ways to find beatmaps to mod, and these are some of the main ones

jumping right into modding queues makes it seem like the whole section is going to be about modding queues